warehouse/admin/templates/admin/organization_applications/detail.html

```diff
@@ -277,10 +277,12 @@ <h4 class="modal-title" id="declineModalLabel">
       </div>
     </div>
 
+    {% set information_requests = organization_application.information_requests %}
+    {% set outstanding_information_requests = organization_application.information_requests|selectattr("additional", "defined")|map(attribute="additional")|selectattr("response", "undefined")|list %}
```

Comment:
question: does the selectattr... and following commands work if `information_requests` is empty? I ask, as it's used as an `if information_requests` conditional later on. Same question applied for the pattern used in `list.html`

Reply:
yep works for empty information_requests